.github/workflows/push.yml

```diff
@@ -46,3 +62,28 @@ jobs:
       run: |
         docker login -u ${{ secrets.DOCKER_USER }} -p ${{ secrets.DOCKER_PASSWORD }}
         ./hack/docker-push.sh
+
+    # actions/checkout MUST come before auth
+    - uses: 'actions/checkout@v2'
+
+    # auth to GCP with OIDC token from GCP Workload Identity Provider. Output auth result to access token.
+    - id: 'auth'
```

Comment:
how about use a long lived credential so we do not need to do the token exchange?
https://cloud.google.com/container-registry/docs/advanced-authentication#json-key

Reply:
It should work too. I will check.